uk.co.compucorp.civicrm.hrleaveandabsences/tests/phpunit/CRM/HRLeaveAndAbsences/Service/LeaveManagerTest.php

```diff
@@ -24,6 +24,48 @@ public function tearDown() {
     $this->unregisterCurrentLoggedInContactFromSession();
   }
 
+  public function testIsContactManagedBy() {
```

Comment:
What about another test/assertion for this but with valid dates set rather than the default null dates set in the `setContactAsLeaveApproverOf` method for the relationship?
i.e using the `setContactAsLeaveApproverOf` method but with valid dates.